### DISCUSSION TITLE
Temperature

### DISCUSSION THREAD

**Discussion by FredrikJohannesson** [DaxStudio/DaxStudio#956] [Q&A] [answered]
Hi!
What is the Temperature column in Vertipaq Analyzer show?

Regards
Fredrik

**Comment by marcosqlbi** [comment, marked as answer]
It's an undocumented value that describes how "hot" a column is - where "hot" means that it is used often in queries, with a time-window analysis of minutes/hours. This tool should help the engine to evaluate which columns to offload from memory.
